styles/template.css

```diff
@@ -0,0 +1,107 @@
+/* Start of Header and Footer Styling */
+.nav-links a,
+.footer-links a {
+    color: inherit;
+    text-decoration: none;
+}
+
+#standard-nav ul {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    padding: 0 50px;
+    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
+}
+
+#city-title {
+    font-weight: bold;
+    font-size: 40px;
+    width: 70%;
+}
+
+.nav-links {
+    font-size: 20px;
+    display: flex;
+    align-items: center;
+}
+
+.nav-links:hover {
+    font-size: bold;
+    border-bottom: 5px solid black;
+    cursor: pointer;
+    font-weight: 700;
+}
+
+.poppins-regular {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+
+body {
+    /* font-family: Arial, sans-serif; */
+    background-color: #FDFFFC;
+    margin: 0;
+    padding: 0;
+}
+
+main {
+    background-image: url("../assets/austin-photos/h-kerry-chipman-PCDtXuVtqYQ-unsplash Large.jpeg");
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    text-align: center;
+    margin: auto;
+    padding-bottom: 100px;
+}
+
+h1 {
+    text-align: center;
+    padding: 20px;
+    text-shadow: 2px 2px 2px rgba(0, 0, 0, 0.2);
+    margin-top: 0;
+    font-size: 2em;
+    background-color: rgba(250, 235, 215, 0.483);
+}
+
+#footer-section-one, #footer-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    color: #FCFFFC;
+}
+
+#footer-section-one {
+    background-color: #151515;
+}
+#footer-text {
+    background-color: #000000;
+}
+
+#footer-section-one ul {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.footer-links {
+    padding: 25px 10vw;
+}
+
+.footer-links .material-symbols-outlined {
+    font-size: 50px;
+    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
+}
+
+.footer-links .material-symbols-outlined:hover {
+    transform: scale(1.2);
+    color: lightgrey;
+}
+
+#footer-text {
+    padding: 25px 0;
+    margin: 0;
+}
+/* End of Header and Footer Styling */
```

Comment:
Don't forget that if you have some styling that will be applied across a number of pages you could always just make a general style sheet and link it to the necessary pages.